ci-operator/step-registry/sdn/OWNERS

```diff
@@ -0,0 +1,17 @@
+# DO NOT EDIT; this file is auto-generated using https://github.com/openshift/ci-tools.
```

Comment:
nit: we'll have to drop this `DO NOT EDIT` line (and others like it throughout this commit), because the ci-tools tooling is not managing the `OWNERS` under `step-registry`.  I'd like to have the tooling maintain per-team aliases (rolling back 0f475599e5, #2572 ) so we could use the aliases in these files.  But until that happens, these `OWNERS` have to be maintained by hand.

Reply:
OK